hera_filters/dspec.py

```diff
@@ -2809,3 +2809,219 @@ def dayenu_mat_inv(x, filter_centers, filter_half_widths,
     else:
         sdwi_mat = cache[filter_key]
     return sdwi_mat
+
+def _kron_matvec(
+    x: np.ndarray,
+    weights: np.ndarray,
+    axis_1_basis: np.ndarray,
+    axis_2_basis: np.ndarray
+)-> np.ndarray:
+    """
+    Compute the matrix-vector product (Kronecker structured) for a given vector.
+
+    Given axis_1_basis (m x i) and axis_2_basis (n x j), this function computes:
+        Y = axis_1_basis * X * axis_2_basis^T
```

Comment:
This isn't actually what you return, you should document how the weights are used like you do in _kron_rmatvec

Reply:
Fixed the documentation here